### DISCUSSION TITLE
SPAWNERMETA COMPATIBILITY

### DISCUSSION THREAD

**Discussion by tomimza** [BG-Software-LLC/SuperiorSkyblock2#1882] [Pending Feature Requests]
Could you make spawnermeta compatible with superiorskyblock2? I consider the spawnermeta plugin to be very good and it would be a plugin which, if it is compatible with superiorskyblock, many people would agree with. I hope they make it compatible soon :)

**Comment by OmerBenGera** [comment]
How is it not compatible?

**Reply by tomimza** [reply]
What happens is that, for example, when I look at /is top and touch a player's head. I see the statistics of the island, the spawners there are, etc. So there is the problem, when you add a spawner, the superiorskyblock plugin does not take the spawner, so it does not add up as statistics.

**Reply by OmerBenGera** [reply]
Sounds like SpawnerMeta should add support to SSB by calling Island#handleBlockPlace or by manually calling the BlockPlaceEvent.

**Reply by tomimza** [reply]
I talked to the creator of spawnermeta and he will add compatibility. Thank you very much for answering!
